dotnet/README.md

```diff
@@ -4,13 +4,20 @@ Tool to automatically generate a `flatpak-builder` sources file from a .NET Core
 
 ## Requirements
 
-You need to have `org.freedesktop.Sdk` and `org.freedesktop.Sdk.Extension.dotnet` installed,
-both branch 18.08.
+You need to have `org.freedesktop.Sdk` and `org.freedesktop.Sdk.Extension.dotnet` installed.
+both branch 21.08 (22.08 is selectable via the `--dotnet` or `-d` arguments)
 
 ## Usage
 
-Run `flatpak-dotnet-generator.py my-output-sources.json my.input.Desktop.csproj`. Then,
-you can use the sources file like this:
+Run `flatpak-dotnet-generator.py my-output-sources.json my.input.Desktop.csproj`. 
+
+### Arguments
+- `--runtime` or `-r` The target [runtime](https://learn.microsoft.com/en-us/dotnet/core/rid-catalog#linux-rids) to restore packages for.
+- `--dotnet` or `-d` The target version of dotnet to use. (5,6,7 are valid.)
+- `--freedesktop` or `-f` The target version of the freedesktop sdk to use. (21.08 by default, you can select 22.08 if wanted.)
+- `--destdir` The directory the generated sources file will save sources to `nuget-sources` by default. 
+
+Then, you can use the sources file like this:
```

Comment:
This part should probably be under Usage still, the placement here feels odd.

Reply:
Agreed, at second glance this did look off.

Reply:
fixed as of 76b360d22d478738e4c2d450ec883a710c3bea37.